### ISSUE TITLE
Multi-worker support

### ISSUE THREAD

**Issue by eadasiak** [SumoLogic/fluentd-output-sumologic#30]
It doesn't appear that this plugin supports the multi-worker configuration using the latest version of Fluentd 1.2.2 (via `td-agent v3.2.0`):

```
error_class=Fluent::ConfigError error="Plugin 'sumologic' does not support multi workers configuration (Fluent::Plugin::Sumologic)"
```
https://docs.fluentd.org/v1.0/articles/performance-tuning-single-process#multi-workers

The details on the changes required are in this blog post:  https://www.fluentd.org/blog/fluentd-v0.14.12-has-been-released

**Comment by frankreno** [comment]
Hi @eadasiak, thanks for the issue.  We will work on adding this capability in the future, but I cannot say when.  Is this currently blocking you? We would welcome a PR if you want to take a stab at it.

**Comment by frankreno** [comment]
This has been fixed in version 1.2.0...
